x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -647,12 +650,15 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
         selectionMode={selectionMode}
         currentQuery={kuery}
         selectedAgents={selectedAgents}
-        refreshAgents={({ refreshTags = false }: { refreshTags?: boolean } = {}) =>
-          Promise.all([fetchData({ refreshTags }), refreshUpgrades()])
-        }
+        refreshAgents={({ refreshTags = false }: { refreshTags?: boolean } = {}) => {
+          Promise.all([fetchData({ refreshTags })]);
+          setShowAgentActivityTour({ isOpen: true });
+        }}
         onClickAddAgent={() => setEnrollmentFlyoutState({ isOpen: true })}
         onClickAddFleetServer={onClickAddFleetServer}
         visibleAgents={agents}
+        onClickAgentActivity={onClickAgentActivity}
+        showAgentActivityTour={showAgentActivityTour}
```

Comment:
One future refactor I think that would be beneficial in the future would be moving all of our flyouts like this to React context objects and placing the `EuiPortal`s all in one spot. 

Related to https://github.com/elastic/kibana/issues/131839, but getting away from some of these many-propped components and just opening/closing flyouts through hooks would be a welcome improvement I think. 

Something for the future, not in this PR.

Reply:
Initially I wanted to put this flyout into the `flyout_context`, but I got stuck on how to pass a callback to it that is outside of the scope of the context. I didn't find any documentation or examples yet on how to do that.
```
<AgentActivityFlyout
            onAbortSuccess={fetchData}
```